spec/features/02_user_sign_in_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature "user signs into website" do
+  let!(:user_1) { User.create(email: 'jscho13@gmail.com',
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.